src/reporter/tests/run_tests.sh

```diff
@@ -6,7 +6,6 @@ sleep 20
 cd ../../../
 pytest src/reporter/ \
        --cov-report= --cov-config=.coveragerc --cov-append --cov=src/ \
-       --ignore=src/reporter/tests/test_health.py \
```

Comment:
this change is already in #449, so perhaps we should merge #449 first and then double check what happens when rebasing, i.e. make sure nothing gets undone!